pkg/test/harness.go

```diff
@@ -253,51 +255,54 @@ func (h *Harness) Config() (*rest.Config, error) {
 			return nil, err
 		}
 		h.config.WarningHandler = rest.NewWarningWriter(os.Stderr, rest.WarningWriterOptions{Deduplicate: true})
-		inCluster, err := testutils.InClusterConfig()
```

Comment:
I'm most concern with this removal... but it is likely based on code review the case of the issue you are looking to solve.   I vaguely remember needing this to satisfy the way operator SDK scorecard uses kuttl.